frontends/mit-open/src/pages/HomePage/FeaturedResourcesSection.tsx

```diff
@@ -12,34 +12,41 @@ const Section = styled.section`
   }
 `
 
-const SORT_AND_PAGE: FeaturedListParams = {
+const COMMON_PARAMS: FeaturedListParams = {
   resource_type: ["course"],
   limit: 12,
-  sortby: "upcoming",
```

Comment:
The `/featured/` view has a sort builtin to ensure balanced listing across offerors.